elide-contrib/elide-dynamic-config-helpers/src/main/resources/templates/security.hbs

```diff
@@ -8,10 +8,10 @@ package dynamicconfig.models;
 import com.yahoo.elide.annotation.SecurityCheck;
 import com.yahoo.elide.security.checks.prefab.Role.RoleMemberCheck;
 
-@SecurityCheck(DynamicConfigOperationChecksPrincipalIs<%#titleCaseRemoveSpaces this%><%/titleCaseRemoveSpaces%>.PRINCIPAL_IS_<%#toUpperCase this%><%/toUpperCase%>)
+@SecurityCheck(DynamicConfigOperationChecksPrincipalIs<%#titleCaseRemoveSpaces this%><%/titleCaseRemoveSpaces%>.PRINCIPAL_IS_<%#toUpperCaseReplaceSpace this%><%/toUpperCaseReplaceSpace%>)
```

Comment:
Does Handlebar allow to chain functions? would something like this work 
`<%#replaceSpace<%#toUpperCase this%><%/toUpperCase%>%><%/replaceSpace%>`

Reply:
I gave it a try but doesn't seem to working.

Reply:
okie.